website/docs/examples/track_planner/README.md

```diff
@@ -0,0 +1,107 @@
+---
+id: track-planner
+title: Track Planner
+---
+
+:::info Basic Knowledge Requirements
+
+Before diving into this code, here's a quick heads-up on what you'll need to be familiar with:
+
+1. **Python Programming**: It's important to have a good grasp of Python, especially with concepts
+like `functions`, `loops`, and `classes`, since the example utilizes these fundamentals.
+2. **Asynchronous Programming with asyncio**: Familiarity with Python's asyncio for writing concurrent
+code using the `async/await` syntax.
+3. **[farm-ng Filter Service Overview](/docs/concepts/filter_service/)**:
+This overview provides a base understanding of the gRPC service the client you create will connect to.
+4. [**farm-ng Transforms & Poses Overview**](/docs/concepts/transforms_and_poses/):
+This overview provides insight into coordinate frames, transforms,
+and poses as they pertain to autonomous systems and autonomous navigation.
+5. [**farm-ng Tracks & Waypoints Overview**](/docs/concepts/tracks_and_waypoints/):
+This overview provides insight into compiling poses as waypoints into a Track
+that can be followed by the Amiga.
+:::
+
+The [**Track Planner Example**](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/track_planner/main.py)
+operates as a standalone Python script,
+in which a [**`Track`**](https://github.com/farm-ng/farm-ng-amiga/blob/main/protos/farm_ng/track/track.proto)
+proto message is generated using the [TrackPlanner](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/track_planner/track_planner.py#L27)
```

Comment:
line # is subject to change when pointed to main. If you pin it to a SHA you can put line #, but then clicking the link may not take them to the latest version. I typically just link to the file because I don't know a better way.
```suggestion
proto message is generated using the [TrackPlanner](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/track_planner/track_planner.py)
```